core/controllers/base.py

```diff
@@ -351,6 +351,9 @@ def render_template(
         if 'nav_mode' not in values:
             values['nav_mode'] = ''
 
+        if 'on_localhost' not in values:
```

Comment:
Instead of using a new variable on_localhost, can we just use the existing DEV_MODE instead?

Reply:
Hi @seanlip  , as you said DEV_MODE works too . So should i change it ?

Reply:
Yes, please. As a general rule, the fewer changes there are to existing code, the better.